diagrams/request_uri_mode_post.md

```diff
@@ -0,0 +1,38 @@
+```plantuml
+@startuml
+
+autonumber
+
+participant "Wallet" as w
+
+participant "User Agent" as u
+
+participant "Verifier" as r
+
+u --> r : use
+activate r
+
+r --> u: authorization request\n(client_id, request_uri, request_uri_method=post, [client_id_scheme])
+deactivate r
+u --> w: authorization request\n(client_id, request_uri, request_uri_method=post, [client_id_scheme])
+activate w
+w --> r: POST **request_uri** (\n[OPTIONAL]wallet_metadata, \n[OPTIONAL]wallet_nonce)
```

Comment:
Just nitpicking here...
```suggestion
w --> r: POST **request_uri** (\n[wallet_metadata][, wallet_nonce])
```

Reply:
both parameters are optional

Reply:
Optional parameters were set into brackets in previous messages.